docs/DEVELOPMENT.md

```diff
@@ -0,0 +1,228 @@
+# Development Guide
+
+This guide covers the development setup and workflow for contributing to the UNDRR Mangrove component library.
+
+## Prerequisites
+
+- Node.js 22 (use nvm or similar to manage versions)
+- Yarn v4 (enabled via Corepack)
+- Docker (optional, for containerized development)
+- Git
+
+## Getting Started
+
+### Local Development
+
+```bash
+# Clone the repository
+git clone git@github.com:unisdr/undrr-mangrove.git
+cd undrr-mangrove
+
+# Enable Corepack for Yarn v4
+corepack enable
+
+# Install dependencies
+yarn install
+
+# Start Storybook development server
+yarn dev
+# or
+yarn storybook
+```
+
+### Docker Development
+
+For consistent environments or Windows development:
+
+```bash
+# Start containers and run Storybook
+yarn docker-run
+
+# Or use Make commands
+make up
+make install
+make run
+```
+
+## Development Workflow
```

Comment:
I think it makes sense to keep this guidance (development) in the readme.md, but we should cross link with the development guidance at https://unisdr.github.io/undrr-mangrove/?path=/docs/getting-started-a-getting-started-guide--docs#development-setup and note that standards on git conventions and more detailed development information are in the readme.md